src/test/fixtures/profiles/timings-with-js.js

```diff
@@ -76,8 +76,8 @@ const thread = {
 };
 
 const profile = {
-  meta: cloneDeep(exampleProfile.meta),
-  libs: cloneDeep(exampleProfile),
+  meta: exampleProfile().meta,
+  libs: exampleProfile(),
```

Comment:
nit: call exampleProfile() once and reuse it in meta and libs.

Looking at this, I'm quite sure that `libs` should be `exampleProfile.libs` :)